adapters/openx/openx_test.go

```diff
@@ -33,6 +34,68 @@ func TestResponseWithCurrencies(t *testing.T) {
 	assertCurrencyInBidResponse(t, "EUR", &currency)
 }
 
+func TestGetBidMeta(t *testing.T) {
+	buyerId := "123"
+	dspId := "456"
+	bradId := "789"
+
+	testCases := []struct {
+		ext          *oxBidExt
+		expectedMeta *openrtb_ext.ExtBidPrebidMeta
+	}{
+		{
+			&oxBidExt{BuyerId: &buyerId, DspId: &dspId, BrandId: &bradId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 123, NetworkID: 456, BrandID: 789},
+		},
+		{
+			&oxBidExt{BuyerId: &buyerId, DspId: &dspId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 123, NetworkID: 456, BrandID: 0},
+		},
+		{
+			&oxBidExt{BuyerId: &buyerId, BrandId: &bradId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 123, NetworkID: 0, BrandID: 789},
+		},
+		{
+			&oxBidExt{DspId: &dspId, BrandId: &bradId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 0, NetworkID: 456, BrandID: 789},
+		},
+		{
+			&oxBidExt{BuyerId: &buyerId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 123, NetworkID: 0, BrandID: 0},
+		},
+		{
+			&oxBidExt{DspId: &dspId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 0, NetworkID: 456, BrandID: 0},
+		},
+		{
+			&oxBidExt{BrandId: &bradId},
+			&openrtb_ext.ExtBidPrebidMeta{AdvertiserID: 0, NetworkID: 0, BrandID: 789},
+		},
+	}
+
+	for _, testCase := range testCases {
+		marshaledExt, _ := json.Marshal(testCase.ext)
+		bid := &openrtb2.Bid{Ext: marshaledExt}
+		updatedMeta := getBidMeta(bid)
+		assert.Equal(t, testCase.expectedMeta, updatedMeta)
+	}
+}
+
+func TestOpenxAdapter_MakeBids_BidsMeta(t *testing.T) {
```

Comment:
What do you think about verifying whole TypedBid objects returned from MakeBids function, not only BidMeta?

Reply:
I refactored this, hope now it's everything correct :D

Reply:
looks good :) Just update the test name please:
```suggestion
func TestOpenxAdapter_MakeBids(t *testing.T) {
```